HelloWorld.python/README.md

```diff
@@ -0,0 +1,21 @@
+<img src="/images/malogo.png" width="300" align="right" /><br><br><br>
+
+# McLaren Applied **ATLAS Automation API Python Sample Code**.
+
+1. Imports modules and stores paths (1-51)
+2. Opens Atlas and confirms connection (54-71)
+3. Loads sets and sessions (74-85)
+4. Creates a range of variables needed (88-111)
+5. Creates a transient parameter (113-131)
+6. Creates waveform (134-143)
+7. Displays parameters in waveform (146-163)
+8. Closes clients (166-170)
+
+This program is a sample of how to control an instance Atlas that is currently running externally through the APIs available with Python.
+
+**Instructions**: Before running, start ATLAS, load a session<sup>1</sup> with parameter `vCar` in it and execute the sample application<sup>2</sup>. 
```

Comment:
You should add the command for people to run to install all the dependencies in the Requirements.txt file.